src/Defs.h

```diff
@@ -89,14 +89,15 @@ typedef uint16_t in_port_t;
 #include <errno.h>
 #include <fcntl.h>
 #include <sys/types.h>		/* sockets*/
+#include <queue>
 
 #include "Ticks.h"
 #include "Message.h"
 #include "Playback.h"
 
 #if ! defined (WIN32) && ! defined (__FreeBSD__)
 	#include "vma-redirect.h"
-	//#define USING_VMA_EXTRA_API
+	#define USING_VMA_EXTRA_API
```

Comment:
need to have ./configure addition to detect vma_extra.h and only then enable the `USING_VMA_EXTRA_API`